spec/spec_helper.rb

```diff
@@ -1,4 +1,5 @@
 require 'support/database_cleaner.rb'
+require 'devise'
```

Comment:
This one seems to be needed.  This link on the [Performatec Wiki for Devise](https://github.com/plataformatec/devise/wiki/How-To:-Test-controllers-with-Rails-3-and-4-(and-RSpec)#controller-specs) suggests it.